tests/test_commands.py

```diff
@@ -443,6 +444,109 @@ def test_update_unit_sto4_start_time_exists_mock_prompt_True(
     ]
 
 
+# update_unit_quota
+
+
+def test_update_unit_no_such_unit(client, runner, capfd: LogCaptureFixture) -> None:
```

Comment:
We need to rename the tests, they are currently duplicates --> only one of the tests will run. 
The previous one --> `test_update_unit_sto4_no_such_unit`
The new one --> `test_update_unit_quota_no_such_unit`

And the same for the other tests below where there are "duplicates"